ccd-definition/ComplexTypes/CareSupervision/2_ChildParty.json

```diff
@@ -158,131 +168,120 @@
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "ChildParty",
     "ListElementCode": "mothersName",
     "FieldType": "Text",
-    "ElementLabel": "Mother's full name",
+    "ElementLabel": "Birth mother's full name",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "ChildParty",
     "ListElementCode": "fathersName",
     "FieldType": "Text",
-    "ElementLabel": "Father's full name",
+    "ElementLabel": "Birth father's full name",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "ChildParty",
-    "ListElementCode": "fathersResponsibility",
-    "FieldType": "FixedList",
-    "FieldTypeParameter": "FathersResponsibilityList",
-    "ElementLabel": "Does the father have parental responsibility?",
+    "ListElementCode": "adoption",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Is adoption being considered at this stage?",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "ChildParty",
-    "ListElementCode": "socialWorkerName",
-    "FieldType": "Text",
-    "ElementLabel": "Name of social worker",
+    "ListElementCode": "placementOrderApplication",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Are you submitting an application for a placement order?",
+    "FieldShowCondition": "adoption=\"Yes\"",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "ChildParty",
-    "ListElementCode": "socialWorkerTelephoneNumber",
-    "FieldType": "Telephone",
-    "ElementLabel": "Social worker's telephone number",
+    "ListElementCode": "placementCourt",
+    "FieldType": "TextArea",
+    "ElementLabel": "Which court are you applying to?",
+    "FieldShowCondition": "placementOrderApplication=\"Yes\" AND adoption=\"Yes\"",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "ChildParty",
-    "ListElementCode": "additionalNeeds",
-    "FieldType": "YesOrNo",
-    "ElementLabel": "Does the child have any additional needs?",
-    "HintText": "For example, child has severe autism and learning disabilities. Special needs for autism should be taken into account.",
+    "ListElementCode": "fathersResponsibility",
+    "FieldType": "FixedList",
+    "FieldTypeParameter": "FathersResponsibilityList",
+    "ElementLabel": "Does the father have parental responsibility?",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "ChildParty",
-    "ListElementCode": "additionalNeedsDetails",
-    "FieldType": "TextArea",
-    "ElementLabel": "Give details",
-    "FieldShowCondition": "additionalNeeds=\"Yes\"",
+    "ListElementCode": "socialWorkerName",
+    "FieldType": "Text",
+    "ElementLabel": "Name of social worker",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "ChildParty",
-    "ListElementCode": "detailsHidden",
```

Comment:
it looks like this field has been totally removed - this would break CCD validation, no fields can be _fully_ removed, just made do_not_show and deprecated, otherwise existing data will break. Worth doing a double check to confirm all "removed" fields aren't actually removed (have tried to do a check which is where this one came from!)

Reply:
detailsHiddenReason, proceedingsLabel, litigationIssues, litigationIssuesDetails too